v2.1/change-data-capture.md

```diff
@@ -7,29 +7,61 @@ toc: true
 <span class="version-tag">New in v2.1:</span> Change data capture (CDC) provides efficient, distributed, row-level change feeds into Apache Kafka for downstream processing such as reporting, caching, or full-text indexing.
 
 {{site.data.alerts.callout_danger}}
-**This feature is under active development** and only works for a targeted use case. Please [file a Github issue](file-an-issue.html) if you have feedback on the interface.
-
-In v2.1, CDC will be an enterprise feature and will have a core version.
+**This feature is under active development** and only works for a targeted use case. Please [file a Github issue](file-an-issue.html) if you have feedback on the roadmap.
 {{site.data.alerts.end}}
 
+{{site.data.alerts.callout_info}}
+CDC is an enterprise feature. There will be a core version in v2.2.
```

Comment:
Do we generally commit to when future versions are coming, or do we want to say "a core version is planned for a future release"?